examples/hyrax.yaml

```diff
@@ -7,597 +9,600 @@
 
 # Administrative information about the profile/model
 
-#    profile_responsibility: No URI available
-    profile_responsibility statement: Samvera Community
-    profile_date_modified: 2019-04-12
-    profile_type: concept #?
-    profile_version: 0.1
+# profile_responsibility: No URI available
+profile_responsibility_statement: Samvera Community
+profile_date_modified: 2019-04-12
+profile_type: concept #?
+profile_version: 0.1
 
 # Property Definition
+properties:
+  - property_method_name: creator
```

Comment:
Why are we using `property_method_name` here instead of 
```yaml
- creator:
    display_label: "Creator"
```
and so on?

Reply:
I think we are, but that file is an older out of date version, if you look at the example on the hyrax-update branch it's very different.